File: Praktikum 1/BunchOfKeys/BunchOfKeys.cpp
#include "BunchOfKeys.hpp"

using namespace std;

BunchOfKeys::BunchOfKeys() {
    this->n_keys = 0;
}

void BunchOfKeys::add() {
    n_keys++;
}

void BunchOfKeys::shake() {
    if (this->n_keys <= 1) {
        cout << "Tidak terjadi apa-apa" << endl;
    } else {
        for (int i = 0; i < n_keys; i++) {
            cout << "krincing" << endl;
        }
    }
}

File: Praktikum 4/BunchOfKeys.java
class BunchOfKeys {
    private int n_keys;

    public BunchOfKeys() {
        this.n_keys = 0;
    }

    public void add() {
        this.n_keys += 1;
    }

    public void shake() {
        if (this.n_keys <= 1) {
            System.out.println("Tidak terjadi apa-apa");
        } else {
            for (int i = 0; i < n_keys; i++) {
                System.out.println("krincing");
            }
        }
    }
}